src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionProvider.cs

```diff
@@ -152,8 +152,13 @@ async Task<ImmutableArray<CompletionItem>> HandleReferenceAsync(IAssemblySymbol
                 // Skip reference with only non-global alias.
                 var metadataReference = compilation.GetMetadataReference(referencedAssemblySymbol);
 
-                if (metadataReference.Properties.Aliases.IsEmpty ||
-                    metadataReference.Properties.Aliases.Any(alias => alias == MetadataReferenceProperties.GlobalAlias))
+                // metadataReference can be null for script compilation, because compilations of previous 
+                // submissions are treated as referenced assemblies. We don't need to check for unimported
```

Comment:
```suggestion
                // submissions aren't treated as referenced assemblies. We don't need to check for unimported
```

Is this correct? Because previous submissions are linked via the Compilation chain versus a regular reference?

Reply:
I guess I wasn't clear in the comment. This is what I'm taking about:
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/Shared/Extensions/ICompilationExtensions.cs,41

The helper I used does this.

Reply:
Perhaps we should use 
`compilation.Assembly.Modules.First().ReferencedAssemblySymbols` instead since the extension method does not add any value here?

Reply:
I'd actually prefer to keep using the helper since it's not obvious what that line of code does. So it might be better to keep it at a single place with proper comment rather than copying the same comments all over.

Reply:
Since we had a null ref here, it's apparently not obvious what ` GetReferencedAssemblySymbols` actually does either :)
So perhaps we should add a parameter `includePreviousSubmissions` that the caller specifies.

Reply:
Sounds good, will do